.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -37,6 +37,15 @@ body:
       label: "🎲 Appwrite version"
       description: "What version of Appwrite are you running?"
       options:
+        - Version 1.3.x
+        - Version 1.2.x
+        - Version 1.1.x
+        - Version 1.0.x      
+        - Version 0.15.x
+        - Version 0.14.x
+        - Version 0.13.x
+        - Version 0.12.x
+        - Version 0.11.x      
```

Comment:
This file looks pretty generic. Can we maybe delete this file from this repo and have it use the default from https://github.com/appwrite/.github/blob/main/.github/ISSUE_TEMPLATE/bug.yaml?

Reply:
Sounds like a good idea
Should we remove these altogether from here, then?

Reply:
@stnguyen90 is that a good next step for now?

Reply:
@stnguyen90 lemme know whatever you think is a good next step here